.gitignore

```diff
@@ -27,7 +27,6 @@ var/
 #  Usually these files are written by a python script from a template
 #  before PyInstaller builds the exe, so as to inject date/other infos into it.
 *.manifest
-*.spec
```

Comment:
The *.spec file is indeed somehow a temporary intermediate file from PyInstaller but according to their documentation it can also be used for configuring the application build. The file gets created whether you use it or not because you can also build the app like this: 

`pyinstaller --hidden-module=... --hidden-module ... --add-data src/res`

This step automatically creates a spec file and initiates a build. You can read more on PyInstaller spec files [here](https://pyinstaller.readthedocs.io/en/stable/spec-files.html).

I was thinking that our dynamic module import system is not very transparent for a developer or an end user and it will infact get worse when we add more functionality to instantshare because, as of right now, we would be adding that functionality by adding more modules that will be dynamically imported. This means that, as the app grows, it might get kind of confusing which modules are hidden imports and which are not. I tried conquering that complexity by basing our application build process on the PyInstaller spec file that contains a maintained list of every dynamically imported module in our app but I overlooked that it contains some information specific to the system it was running on when the spec file was created (i.e. file paths). 

The solution should be simple though because I think that the parameter `pathex` that contains the system specific path can be safely deleted (just tested this in Windows). Please test building instantshare with the new commit.